tensorrt_llm/executor/worker.py

```diff
@@ -595,6 +601,7 @@ def worker_main(
     is_llm_executor: Optional[
         bool] = True,  # whether it's the main executor instance
     lora_config: Optional[LoraConfig] = None,
+    garbage_collection_gen0_threshold: Optional[int] = None,
```

Comment:
How about changing the typing to `int` since it gets a value of 20000 as the default value.

Reply:
Yeah, we can keep the Optional to allow user specify None.